shell/server/pre-meteor.js

```diff
@@ -127,6 +131,13 @@ function wwwHandlerForGrain(grainId) {
           });
           response.end("redirect: /" + path + "/");
         } else if (status === "notFound") {
+          if (path !== encoded) {
+            // Compatibility hack: Try geting the URL-encoded version of the path. We used to not
```

Comment:
Actually "getting" is what I intended, as in we're "getting" the file from the grain.

Though I meant it with two t's...